dist/profile/templates/confluence/vhost.conf

```diff
@@ -1609,6 +1609,7 @@ RewriteRule "^/display/JENKINS/Pipeline\+SCM\+Step\+Plugin$" "https://plugins.je
 RewriteRule "^/display/JENKINS/Step\+by\+step\+guide\+to\+set\+up\+master\+and\+agent\+machines\+on\+Windows$" "https://www.jenkins.io/doc/book/using/using-agents/" [NE,NC,L,QSA,R=301]
 RewriteRule "^/display/JENKINS/Search\+Box$" "https://www.jenkins.io/doc/book/using/searchbox/" [NE,NC,L,QSA,R=301]
 RewriteRule "^/display/JENKINS/I'm\+getting\+OutOfMemoryError$" "https://www.jenkins.io/doc/book/system-administration/diagnosing-errors#out-of-memory-error" [NE,NC,L,QSA,R=301]
+RewriteRule "^display/JENKINS/Quick+and+Simple+Security$" "https://www.jenkins.io/doc/book/installing/#war-file" [NE,NC,L,QSA,R=301]
```

Comment:
Thanks for the pull request.  I proposed a few changes:

* needs the leading '/' to match the URL
* Needs to escape the '+' character in the left hand side of the rule
* Use the final destination https://www.jenkins.io/doc/book/installing/war-file/ instead of the previous destination https://www.jenkins.io/doc/book/installing/#war-file so that we avoid an additional redirect
```suggestion
RewriteRule "^/display/JENKINS/Quick\+and\+Simple\+Security$" "https://www.jenkins.io/doc/book/installing/war-file/" [NE,NC,L,QSA,R=301]
```